source/main/physics/BeamFactory.cpp

```diff
@@ -1077,21 +1077,21 @@ void ActorManager::UpdateActors(Actor* player_actor, float dt)
             actor->ar_vehicle_ai->update(dt, 0);
 #endif // USE_ANGELSCRIPT
 
-        if (actor->ar_sim_state == Actor::SimState::LOCAL_SIMULATED)
+        if (actor->ar_sim_state < Actor::SimState::LOCAL_SIMULATED)
         {
             actor->updateVisual(dt);
             actor->updateSkidmarks();
             actor->UpdateFlareStates(dt); // Only state, visuals done by GfxActor
             actor->m_gfx_actor->UpdateParticles(dt); // TODO: move it to GfxActor ~ only_a_ptr, 06/2018
-            if (actor->ar_uses_networking)
+            if (actor->ar_sim_state == Actor::SimState::NETWORKED_OK)

```

Comment:
this is inside `if (ar_sim_state < LOCAL_SIMULATED)`, control can't ever get here.

Reply:
I fucked it up. ^^